### PR TITLE
T3chguy/num members tooltip

### LINKED ISSUE #5858: Hovering over 12K in matrix hq should give you the actual number of users
### Description

image
^ this icon. Hovering it should tell me 12,453 or whatever it is.

### Version information

- **Platform**: web (in-browser)
- **Browser**: Chrome 63
- **OS**: Windows 10
- **URL**: riot.im/develop

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes #5858

### CODE REVIEW

File: src/components/structures/RightPanel.js [hunk 1]
```diff
@@ -266,22 +265,23 @@ module.exports = React.createClass({
...
             const cli = this.context.matrixClient;
             const room = cli.getRoom(this.props.roomId);
             let userIsInRoom;
             if (room) {
-                membersBadge = formatCount(room.getJoinedMembers().length);
-                userIsInRoom = room.hasMembershipState(
-                    this.context.matrixClient.credentials.userId, 'join',
-                );
+                const numMembers = room.getJoinedMembers().length;
+                membersTitle = _t('%(numMembers)s Members', { numMembers });
```
Comment (turt2live) [inline src/components/structures/RightPanel.js]: and if there's one member?

Comment (t3chguy) [reply src/components/structures/RightPanel.js]: Arg

File: src/components/structures/RightPanel.js [hunk 2]
```diff
@@ -266,17 +266,18 @@ module.exports = React.createClass({
...
+                membersBadge = <div title={membersTitle}>{ formatCount(numMembers) }</div>;
+                userIsInRoom = room.hasMembershipState(this.context.matrixClient.credentials.userId, 'join');
```
Comment (lukebarnard1) [inline src/components/structures/RightPanel.js]: I know it's not your fault, but can you rename `userIsInRoom` to `isUserInRoom` - I'm finding it easy to misread.

File: src/components/structures/RightPanel.js [hunk 3]
```diff
@@ -266,22 +265,23 @@ module.exports = React.createClass({
...
             let userIsInRoom;
             if (room) {
-                membersBadge = formatCount(room.getJoinedMembers().length);
-                userIsInRoom = room.hasMembershipState(
-                    this.context.matrixClient.credentials.userId, 'join',
-                );
+                const numMembers = room.getJoinedMembers().length;
+                membersTitle = _t('%(count)s Members', { count: numMembers });
+                membersBadge = <div title={membersTitle}>{ formatCount(numMembers) }</div>;
+                userIsInRoom = room.hasMembershipState(this.context.matrixClient.credentials.userId, 'join');
```
Comment (lukebarnard1) [inline src/components/structures/RightPanel.js]: I know it's not your fault, but can you rename `userIsInRoom` to `isUserInRoom` - I'm finding it easy to misread.

Comment (t3chguy) [reply src/components/structures/RightPanel.js]: done

**Pull request merged**